pkgs/by-name/go/goframe-cli/package.nix

```
@@ -0,0 +1,51 @@
+{
+
+  lib,
+
+  buildGoModule,
+
+  fetchFromGitHub,
+
+}:
+
+buildGoModule rec {
+  pname = "gf";
+  version = "2.9.0";
+
+  src = fetchFromGitHub {
+    owner = "cococolanosugar";
+    repo = "gf";
+    rev = "v${version}";
+    hash = "sha256-o42G1dsz/WM1uI5kIsSZmLaBy9kHl4ZvT73KkB+hf7Y=";
+  };
+
+  subPackages = [ "." ];
+  modRoot = "cmd/gf";
+
+  vendorHash = null;
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
+
+  doCheck = false;
+
+  env = {
+    GOWORK = "off";
+  };
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
Nit: Avoid [wide `with` scopes](https://nix.dev/guides/best-practices#with-scopes) in attribute sets. It's alright to repeat `lib.*` when you need it:
```nix
{
  meta = {
    license = lib.licenses.publicDomain;
    maintainers = with lib.maintainers; [ maintainer ];
  };
}
```